pytorch/training/docker/1.13/py3/example/Dockerfile.sagemaker.lite.gpu

```diff
@@ -0,0 +1,11 @@
+# Expecting base image to be the Deep Learning Container image built by ../cu113/Dockerfile.e3.gpu
+ARG BASE_IMAGE=""
```

Comment:
Please go ahead and remove this dockerfile. We do not need it.

Reply:
The examples are gone.